pom.xml

```diff
@@ -866,19 +878,10 @@
                 <configuration>
                     <redirectTestOutputToFile>${test.redirectOutput}</redirectTestOutputToFile>
                     <argLine>-Xmx512m -DjvmId=${surefire.forkNumber} ${surefire.jacoco.args}</argLine>
-                    <includes>
```

Comment:
integration tests are running during the CI (unit test) build (i.e. run ```mvn clean install```)

Reply:
I will check, on Gitlab CI we use profile release . There are no integration tests running on that. I guess I need to exclude the group in default profile as well

Reply:
Default profile was Junit and integration tests were included in it. I have removed it. I am not sure how it was working before though

Reply:
Confirmed, thanks